cmd/clusterctl/client/move.go

```diff
@@ -113,22 +127,11 @@ func (c *clusterctlClient) Move(options MoveOptions) error {
 }
 
 func (c *clusterctlClient) Backup(options BackupOptions) error {
```

Comment:
This method should be deprecated (same for Restore)
Also, ideally, the implementation should be cleaned up and become a simple call to move.